opentelemetry_api/opentelemetry/context/traceparent.c

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2019, OpenTelemetry Authors
```

Comment:
I would like the implementation to be in C++, not C.

Reply:
I agree. This is C++ SIG and implementations should preferably be in C++.